rfcs/0039-cheque/0039-cheque.md

```diff
@@ -0,0 +1,334 @@
+---
+Number: "0039"
+Category: Standards Track
+Status: Proposal
+Author: Dylan Duan
```

Comment:
Please update metadata to latest format, thanks.

Reply:
Updated. https://github.com/nervosnetwork/rfcs/pull/308/commits/e16494f9f3f3d7a938d6111be2ea90be1fe03e2b